### PR TITLE
Feature/fga/better timeline scroll

### PR DESCRIPTION
**Pull request by ganfra**
This PR provides improvements over : 
- The JumpToBottomButtin look and feel
- The auto-scroll when new message come in 
- The id of `TimelineItem.GroupedEvents`  so it's unique and stable

### CODE REVIEW

**Comment by ganfra** [comment]
@jmartinesp Can you approve or comments if you want some more changes as we know the issues on the timeline are not related?

**Review by jmartinesp — APPROVED** [review]
I wanted to test it manually a bit first, to make sure there are no corner cases we missed, but that won't be possible until the other bug is fixed so I'll approve it and we thoroughly test it later and modify/revert it in the worst case scenario.

Also, a couple of minor comments, feel free to ignore them if you'd rather focus on something else.

**Comment by jmartinesp** [comment]
I just tested it and it works fine. I pushed the contents of the branch after merging the latest `develop` commits and triggered a record screenshots workflow for it which should fix the Test workflow.

**Pull request merged**
